tests/testthat/test-as_html.R

```diff
@@ -36,6 +36,11 @@ test_that("code inside Sexpr is evaluated", {
   expect_equal(rd2html("\\Sexpr{1 + 2}"), "3")
 })
 
+test_that("can convert cross links to online documentation url", {
+  expect_equal(rd2html("\\link[base]{library}"),
+               link_remote(label="library", topic="library", package="base"))
```

Comment:
You have "library" in two places. How will the test detect if the first and second are flipped? Maybe what I'm asking for is also a test that uses the `\link[=label]{function}` form (except I've forgotten exactly what that looks like).

Yup, that's the formatting I prefer.